documentation/_puppetdb_nav.html

```diff
@@ -13,7 +13,7 @@ <h2 id="puppetdb">PuppetDB {{ page.doc.version }}</h2>
   </li>
   <li><strong>Installation</strong>
     <ul>
-      <li><a href="{{puppetdb}}/migrate.html">Migrating data</a></li>
+      <li><a href="{{puppetdb}}/../3.2/migrate.html">Migrating data</a></li>
```

Comment:
The problem with linking this from here, is that it makes it sound like all the instructions in that doc will work on PDB 4, which it actually won't. I'm not quite sure what we're trying to achieve here by having it in the main index. If its upgrading or whatever help that people need in 3.2, perhaps a mention in the release notes is better ... or something like that.